src/parameterizations/vertical/MOM_energetic_PBL.F90

```diff
@@ -978,21 +994,18 @@ subroutine ePBL_column(h, dz, u, v, T0, S0, dSV_dT, dSV_dS, SpV_dt, TKE_forcing,
           MixLen_shape(K) = 1.0
         enddo ; endif
       else
-        ! Reduce the mixing length based on MLD, with a quadratic
-        ! expression that follows KPP.
-        I_MLD = 1.0 / MLD_guess
-        dz_rsum = 0.0
-        MixLen_shape(1) = 1.0
-        do K=2,nz+1
-          dz_rsum = dz_rsum + dz(k-1)
-          if (CS%MixLenExponent==2.0) then
-            MixLen_shape(K) = CS%transLay_scale + (1.0 - CS%transLay_scale) * &
-                 (max(0.0, (MLD_guess - dz_rsum)*I_MLD) )**2 ! CS%MixLenExponent
+
```

Comment:
It looks like you are replacing some existing code, rather than adding a new option. I suspect you need to add `elseif (CS%eqdisc_v0 .or.CS%eqdisc_v0h) then`

Reply:
The subroutines associated with CS%eqdisc_v0 and CS%eqdisc_v0h are different. 
CS%eqdisc_v0 calls the subroutine 'get_eqdisc_v0(CS,absf,B_flux,u_star,v0_dummy)' 
while 
CS%eqdisc_v0h calls 'get_eqdisc_v0h(CS,B_flux,u_star,MLD_guess,v0_dummy)'. 

Both of them formulate v0 in a different manner. 
So it would not be ideal to call them in a single elseif statement.